packages/ra-ui-materialui/src/list/filter/FilterButton.tsx

```diff
@@ -84,6 +84,7 @@ export const FilterButton = (props: FilterButtonProps): JSX.Element => {
                         filter={filterElement}
                         resource={resource}
                         onShow={handleShow}
+                        disabled={filterElement.props.disabled || false}
```

Comment:
I don't understand why you set the disabled prop here, it has no effect?

Reply:
Agreed, on second look this is unnecessary. Will delete this change